layouts/partials/integrations-carousel/integrations-carousel.html

```diff
@@ -0,0 +1,205 @@
+            {{ end }}
+        </div>
+        {{/*  Carousel (vertical)  */}}
+        <div 
+        class="carousel-items__vertical d-none overflow-hidden h-100 d-md-block col-md-3"
+        x-init="setScrollByHeight(); setActiveBtnOffsetTop()"
+        x-ref="verticalCarousel"
+        @resize.window="navigateVerticalNavOnResize()">
+            {{ range $idx, $elm := $media }}
+                {{ if and (gt $idx (sub $asset_len 4)) $isScrollableCarousel}}
+                <button 
+                class="carousel-item__vertical row g-0 p-0 w-100 mx-auto rounded-1"
+                :class="isActiveVertical({{$idx}}, $el)"
+                @click="navigateCarousel(chooseVerticalDirection($el))"
+                :style="{transform: `translateY(-${scrollByHeight * 2}px)`, backgroundImage: 'url(https://s3.amazonaws.com/dd-integrations/{{ $app_id }}/media_gallery/{{ replaceRE `(./)?images/` "" $elm.image_url }})'}"
+                >
+                </button>
+                {{ end }}
+            {{ end }}
+            {{ range $idx, $elm := $media }}
+            <button
+            class="carousel-item__vertical row g-0 p-0 w-100 mx-auto rounded-1"
+            :class="isActiveVertical({{$idx}}, $el)"
+            @click="navigateCarousel(chooseVerticalDirection($el, {{ $idx }}))"
+            :style="{transform: `translateY(-${scrollByHeight * 2}px)`, backgroundImage: 'url(https://s3.amazonaws.com/dd-integrations/{{ $app_id }}/media_gallery/{{ replaceRE `(./)?images/` "" $elm.image_url }})'}"
+            >
+            </button>
+            {{ end }}
+            {{ range $idx, $elm := first 1 $media }}
+                {{ if $isScrollableCarousel}}
+                <button 
+                class="carousel-item__vertical row g-0 p-0 w-100 mx-auto rounded-1"
+                :class="isActiveVertical({{$idx}}, $el)"
+                @click="navigateCarousel(chooseVerticalDirection($el))"
+                :style="{transform: `translateY(-${scrollByHeight * 2}px)`, backgroundImage: 'url(https://s3.amazonaws.com/dd-integrations/{{ $app_id }}/media_gallery/{{ replaceRE `(./)?images/` "" $elm.image_url }})'}"
+                >
+                </button>
+                {{ end }}
+            {{ end }}
+        </div>
+    </div>
+    {{/*  Carousel item description  */}}
+    <div class="carousel-item-descriptions p-0">
+        {{ range $idx, $el := $media }}
+        <div 
+        class="carousel-item-description" 
+        :class="shouldShowActiveHorizontalSlide({{ $idx }}) && 'active'"
+        x-cloak x-show="shouldShowActiveHorizontalSlide({{ $idx }})">
+            <div class="item-counter">
+                <span class="fw-bold" x-text="(activeSlide + 1) + '/{{$asset_len}}'"></span>
+            </div>
+            <div class="item-caption">
+                <p class="m-0">{{$el.caption}}</p>
+            </div>
+        </div>
+        {{ end }}
+    </div>
+</div>
+
+<script src="https://player.vimeo.com/api/player.js"></script>
```

Comment:
should we consider including this player.js somewhere else?

Reply:
Maybe something like https://github.com/DataDog/documentation/blob/master/layouts/partials/header-scripts.html#L65-L72 if eq integrations ?

Reply:
I thought this was appropriate since it's only for controlling the integrations-carousel videos and it would load on the page for only pages where the integration-carousel loads. Is there a reason why we wouldnt keep it here?

Reply:
Just a script tag can be render blocking. I also think the js lib can be easily forgotten when its included this way. Not a huge deal though, lets leave this as is but can you create another card to update after.